### PR TITLE
Fixed issue #5392

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
 #5392

### CODE REVIEW

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue [hunk 1]
```diff
@@ -542,6 +548,8 @@ export default {
                         indexConditions = false;
 
                         alert(this.translations.zeroIndex);
+                    } else {
+                        alert("Please enter the value");
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue:552]: Use translated value.

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue [hunk 2]
```diff
@@ -514,6 +514,10 @@ export default {
         },
 
         getResponse: function() {
+            if (this.type == null) {
+                alert("Please select the filter");
```
Comment (devansh-webkul) [inline packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue:518]: Use translated value.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue (hunk 2 above)
Comment (akathuria289) [reply packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue]: Done

File: packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue (hunk 1 above)
Comment (akathuria289) [reply packages/Webkul/Ui/src/Resources/assets/js/components/datagrid/filters/column-filter.vue]: Done

**Review by jitendra-webkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
**Not working,** please run **_npm run prod_**  and do a fresh PR

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Created PR - https://github.com/bagisto/bagisto/pull/6587

**Pull request merged**